tests/app/views/test_buyers.py

```diff
@@ -1874,6 +1882,396 @@ def test_404_if_lot_does_not_allow_brief(self, data_api_client):
         assert res.status_code == 404
 
 
+class TestDownloadView(BaseApplicationTest):
+    def setup_method(self, method):
+        super(TestDownloadView, self).setup_method(method)
+
+        self.data_api_client = mock.MagicMock(spec_set=DataAPIClient)
+        self.content_loader = mock.MagicMock(spec_set=ContentLoader)
+
+        self.instance = buyers.DownloadView(
+            data_api_client=self.data_api_client,
+            content_loader=self.content_loader
+        )
+
+        self.brief = api_stubs.brief(status='closed')['briefs']
+        self.brief['essentialRequirements'] = [
+            "Good nose for tea",
+            "Good eye for biscuits",
+            "Knowledgable about tea"
+        ]
+        self.brief['niceToHaveRequirements'] = [
+            "Able to bake",
+            "Able to perform the tea ceremony"
+        ]
+        self.brief['blah'] = ['Affirmative', 'Negative']
```

Comment:
What is this for? If it is to signify an unrelated key on the brief then it might be good to name it as such `self.brief['unrelatedKey']` or `self.brief['nonImportantKey']`